src/app/georeferencer/qgsgeorefdatapoint.cpp

```diff
@@ -150,11 +163,13 @@ bool QgsGeorefDataPoint::contains( QPoint p, bool isMapPlugin )
   if ( isMapPlugin )
   {
     QPointF pnt = mGCPSourceItem->mapFromScene( p );
+    QPolygonF bb = mGCPSourceItem->shape().toFillPolygon();
```

Comment:
is this leftover code?

Reply:
Yes, Forgot to remove this, was used for debugging to get the raw coords of the click.